android/res/layout/onmap_downloader.xml

```diff
@@ -16,8 +16,10 @@
     android:clipChildren="false"
     android:layout_gravity="center"
     android:gravity="center_horizontal"
-    android:background="?cardFrame"
+    android:backgroundTint="?menuBackground"
+    android:background="@drawable/card_background"
```

Comment:
Why card_background if it's onmap_downloader_background?

Reply:
renamed 👍